functions/export-csv.js

```diff
@@ -101,6 +104,30 @@ async function exportCsv(req, res) {
   csvStream.end();
 }
 
+function toWkt(geoJsonString) {
+  const geoJsonObject = parseGeoJson(geoJsonString);
+  const geometry = getGeometry(geoJsonObject);
+  return geometry ? wkt.stringify(geometry) : "";
+}
+
+function parseGeoJson(jsonString) {
+  if (!jsonString) {
```

Comment:
nit: JSON.parse(null) gives you null without error, so you can remove this if.

Reply:
Done.